### PR TITLE
New periodic and scheduled effector policies

### PR DESCRIPTION
**Pull request by grkvlt**
Adds policies to execute effectors at specific times, either singly or repeated.

### CODE REVIEW

**Comment by ahgittin** [comment]
cool @grkvlt - wonder if you could do two things:

* merge #818 here for testing and follow the pattern there to set trigger and action highlights
* include a YAML illustrating use of these policies

**Comment by ahgittin** [comment]
(yaml could be used in a test :nudge:)

File: policy/src/main/java/org/apache/brooklyn/policy/action/PeriodicEffectorPolicy.java [hunk 1]
```diff
@@ -0,0 +1,151 @@
...
+ *       period: 1 day
+ *       time: 08:00:00
+ *   - type: org.apache.brooklyn.policy.action.ScheduledEffectorPolicy
+ *     brooklyn.config:
+ *       effector: resize
+ *       args:
+ *         desiredSize: 1
+ *       period: 1 day
+ *       time: 18:00:00
+ * }</pre>
```
Comment (drigodwin) [inline policy/src/main/java/org/apache/brooklyn/policy/action/PeriodicEffectorPolicy.java]: This example is for the `ScheduledEffectorPolicy` not the `PeriodicEffectorPolicy`

File: policy/src/main/java/org/apache/brooklyn/policy/action/PeriodicEffectorPolicy.java [hunk 2]
```diff
@@ -0,0 +1,151 @@
...
+    public void setEntity(final EntityLocal entity) {
+        super.setEntity(entity);
+
+        AttributeSensor<Boolean> sensor = config().get(START_SENSOR);
+        subscriptions().subscribe(entity, sensor, handler);
+    }
+
+    @Override
+    public void rebind() {
+        // XXX Will the AtomicBoolean field be saved during rebind?
```
Comment (drigodwin) [inline policy/src/main/java/org/apache/brooklyn/policy/action/PeriodicEffectorPolicy.java]: Probably resolve this question before merging...

**Comment by drigodwin** [comment]
I have added a PR for the documentation for this [here](https://github.com/apache/brooklyn-docs/pull/213).

**Comment by grkvlt** [comment]
@ahgittin I will do some testing of how this interacts with #818 but it will be in a separate PR so that this can get into the 0.12 RC

**Pull request merged**